src/PrestaShopBundle/Form/Admin/Configure/ShopParameters/ProductPreferences/StockType.php

```diff
@@ -47,59 +47,151 @@ class StockType extends TranslatorAwareType
     public function buildForm(FormBuilderInterface $builder, array $options)
     {
         $builder
-            ->add('allow_ordering_oos', SwitchType::class)
-            ->add('stock_management', SwitchType::class)
+            ->add('stock_management', SwitchType::class, [
+                'label' => $this->trans(
+                    'Enable stock management',
+                    'Admin.Shopparameters.Feature'
+                ),
+                'required' => false,
+            ])
+            ->add('pack_stock_management', ChoiceType::class, [
+                'label' => $this->trans(
+                    'Default pack stock management',
+                    'Admin.Shopparameters.Feature'
+                ),
+                'help' => $this->trans(
+                    'When selling packs of products, how do you want your stock to be calculated?',
+                    'Admin.Shopparameters.Help'
+                ),
+                'choices' => [
+                    'Decrement pack only.' => 0,
+                    'Decrement products in pack only.' => 1,
+                    'Decrement both.' => 2,
```

Comment:
I think these have to be translated

Reply:
The translation is done automatically.

Reply:
How? 🤔 Using which domain?

Reply:
Fixed. Check the key `choice_translation_domain`.